1-datasets-uproot.ipynb

```diff
@@ -35,76 +35,21 @@
-      "f_lept4_phi                (no streamer)              asdtype('>f4')\n",
-      "f_lept4_charge             (no streamer)              asdtype('>f4')\n",
-      "f_lept4_pfx                (no streamer)              asdtype('>f4')\n",
-      "f_lept4_sip                (no streamer)              asdtype('>f4')\n",
-      "f_iso_max                  (no streamer)              asdtype('>f4')\n",
-      "f_sip_max                  (no streamer)              asdtype('>f4')\n",
-      "f_Z1mass                   (no streamer)              asdtype('>f4')\n",
-      "f_Z2mass                   (no streamer)              asdtype('>f4')\n",
-      "f_angle_costhetastar       (no streamer)              asdtype('>f4')\n",
-      "f_angle_costheta1          (no streamer)              asdtype('>f4')\n",
-      "f_angle_costheta2          (no streamer)              asdtype('>f4')\n",
-      "f_angle_phi                (no streamer)              asdtype('>f4')\n",
-      "f_angle_phistar1           (no streamer)              asdtype('>f4')\n",
-      "f_pt4l                     (no streamer)              asdtype('>f4')\n",
-      "f_eta4l                    (no streamer)              asdtype('>f4')\n",
-      "f_mass4l                   (no streamer)              asdtype('>f4')\n",
-      "f_mass4lErr                (no streamer)              asdtype('>f4')\n",
-      "f_njets_pass               (no streamer)              asdtype('>f4')\n",
-      "f_deltajj                  (no streamer)              asdtype('>f4')\n",
-      "f_massjj                   (no streamer)              asdtype('>f4')\n",
-      "f_D_jet                    (no streamer)              asdtype('>f4')\n",
-      "f_jet1_pt                  (no streamer)              asdtype('>f4')\n",
-      "f_jet1_eta                 (no streamer)              asdtype('>f4')\n",
-      "f_jet1_phi                 (no streamer)              asdtype('>f4')\n",
-      "f_jet1_e                   (no streamer)              asdtype('>f4')\n",
-      "f_jet2_pt                  (no streamer)              asdtype('>f4')\n",
-      "f_jet2_eta                 (no streamer)              asdtype('>f4')\n",
-      "f_jet2_phi                 (no streamer)              asdtype('>f4')\n",
-      "f_jet2_e                   (no streamer)              asdtype('>f4')\n",
-      "f_D_bkg_kin                (no streamer)              asdtype('>f4')\n",
-      "f_D_bkg                    (no streamer)              asdtype('>f4')\n",
-      "f_D_gg                     (no streamer)              asdtype('>f4')\n",
-      "f_D_g4                     (no streamer)              asdtype('>f4')\n",
-      "f_Djet_VAJHU               (no streamer)              asdtype('>f4')\n",
-      "f_pfmet                    (no streamer)              asdtype('>f4')\n",
-      "None\n"
+      "--2020-06-20 03:51:56--  https://zenodo.org/record/3901869/files/ntuple_4mu_bkg.root?download=1\n",
```

Comment:
Some default output seems to be missing from here. It's nice to have for when people look at these notebooks on GitHub or don't want to execute the commands themselves. Please add a version with the expected output.

Reply:
Fixed.